docs/backup-recovery/README.md

```diff
@@ -0,0 +1,146 @@
+# Cluster upgrades - backup and recovery #
+
+## Background ##
+
+This feature creates a pre-upgrade backup and provides a procedure for rapid recovery of a SNO in the event of a failed upgrade that was previously seemed unrecoverable. In case of an upgrade failure, this feature allows the SNO to be restored to a working state with the previous version of OCP without requiring a re-provision of the application(s).  
```

Comment:
typo: "previously deemed unrecoverable" ... That said, I still don't think we should be saying "unrecoverable" ... primarily because this feature cannot recover from unrecoverable failures.

Reply:
Removed as suggested